src/writing-policies/go/06-logging.md

```diff
@@ -0,0 +1,58 @@
+# Logging
+
+The Go SDK integrates with the `onelog` project almost out of the box. The reasons why this library
+has been chosen are:
+
+- It works with WebAssembly binaries. Other popular logging solutions cannot even be built to
+  WebAssembly.
+
+- It provides [good performance](https://github.com/francoispqt/onelog#benchmarks).
+
+- It supports structured logging.
+
+## Initialize logger
+
+You first have to initialize a logger structure. By performing this initialization in a global
+variable, you can easily log from the two main policy entry points: `validate` and
+`validate_settings`. Let's initialize this structure in our main package:
+
+```go
+var (
+	logWriter = kubewarden.KubewardenLogWriter{}
+	logger    = onelog.New(
+		&logWriter,
+		onelog.ALL, // shortcut for onelog.DEBUG|onelog.INFO|onelog.WARN|onelog.ERROR|onelog.FATAL
+	)
+)
+```
+
+## Consuming the logger
+
+Now, we can use the `logger` object to log from wherever we need in our policy:
+
+```go
+func validate(payload []byte) ([]byte, error) {
+	// ...
+	logger.Info("validating request")
+	// ...
+}
+```
+
+You can refer to the [`onelog`
+documentation](https://pkg.go.dev/github.com/francoispqt/onelog?utm_source=godoc) for more
+information. Let's add some structured logging:
```

Comment:
I would move the `You can refer.... for more information.` **after** the code block